mk/PcapPlusPlus.mk.linux

```diff
@@ -4,7 +4,7 @@
 PCAPPP_INCLUDES += -I/usr/include/netinet
 
 # libs
-PCAPPP_LIBS += -lpcap -lpthread
+PCAPPP_LIBS += -lpthread -lpcap
```

Comment:
why changing the order?

Reply:
That was a mistake when manually reverting the changes. Will put it back as it was before.